scripts/irods/database_upgrade.py

```diff
@@ -88,6 +88,12 @@ def run_update(irods_config, cursor):
                         "'1580297960');")
         database_connect.execute_sql_statement(cursor, sql)
 
+    elif new_schema_version == 8:
+        if irods_config.catalog_database_type == 'postgres':
+            database_connect.execute_sql_statement(cursor, "alter table R_RULE_EXEC add column exe_context text;")
```

Comment:
we need some discourse/comments here ...

seems like TEXT is good for mysql/mariadb too?   and clob perhaps instead of blob?

Reply:
what would the comments be about?  the data types selected?

based on what i've read about MySQL, TEXT has a max size of 2^16.  LONGTEXT is 4GB.  Which one do we want?